Digital_guj_scraper.py

```diff
@@ -63,9 +31,7 @@ def Sendemail(date,receivers_email):
     msg['Subject'] = 'Latest GTU Notification'
 
     # Create the body of the message (a plain-text and an HTML version).
-    # html = (date + '\n'+Notification+ '\n'+link)
-    html = (date )
-
+    html = (date + '\n'+Notification+ '\n'+link)
```

Comment:
Fix this 

![image](https://github.com/Rjchauhan18/gtu_notification/assets/113838908/d7a40174-0d8c-411c-8e39-af98653bfa86)